src/reactviews/pages/SchemaCompare/components/CompareActionBar.tsx

```diff
@@ -87,6 +87,35 @@ const CompareActionBar = (props: Props) => {
         );
     };
 
+    const disableGenerateScriptButton = (): boolean => {
+        if (
+            !(
+                (
+                    context.state.targetEndpointInfo &&
+                    Number(context.state.targetEndpointInfo.endpointType) === 0
```

Comment:
Better to use the SchemaCompareEndpointType enum instead of a hardcoded number with a comment.